examples/reservoir_simulation/CCUS/set_env.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
license header maybe?

Reply:
Yeah, I have thought about this, I think for shell scripts it will have to be after the shebang line, hence I had not attempted to add it. Will add it after `#!/bin/bash`